Generator/Sources/NeedleFramework/Parsing/Tasks/ASTUtils.swift

```diff
@@ -133,7 +120,7 @@ extension Structure {
     var inheritedTypes: [String] {
         let types = dictionary["key.inheritedtypes"] as? [SourceKitRepresentable] ?? []
         return types.compactMap { (item: SourceKitRepresentable) -> String? in
-            (item as? [String: String])?["key.name"]
+            ((item as? [String: String])?["key.name"])?.replacingOccurrences(of: "\n", with: "").replacingOccurrences(of: " ", with: "")
```

Comment:
nit: This means we can remove the whitespace trimming code elsewhere. We should clean that up later.

Reply:
Done here https://github.com/uber/needle/pull/231